src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -6,13 +6,20 @@
 import org.apache.jena.riot.RDFDataMgr;
 import org.apache.jena.riot.RDFFormat;
 
+import java.io.OutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
 
 import static com.ncc.aif.AIFUtils.*;
 import static org.junit.jupiter.api.Assertions.fail;
 
+import ch.qos.logback.classic.Logger;
```

Comment:
FWIW, we've been using IntelliJ's code formatter for consistent formatting.  One thing it does is organize imports, and it puts this import at the top (it's basically alphabetical).  If you want, I could either push an update with the code formatter changes, send you files with them and you decide what you want, or just tell you what it's changed.  It's not a log, but there are some changes it tries to make.

Reply:
I use Emacs and IntelliJ interchangeably.  I fixed all code using IntelliJ reformat code and optimize imports.